client/packages/invoices/src/Returns/InboundDetailView/Footer/Footer.tsx

```diff
@@ -65,7 +68,11 @@ export const FooterComponent: FC = () => {
           >
             <OnHoldButton />
             <StatusCrumbs
-              statuses={inboundReturnStatuses}
+              statuses={
+                isManuallyCreated
+                  ? manualInboundReturnStatuses
+                  : inboundReturnStatuses
```

Comment:
Oo - do you mean because we would only ever see the automatically created Inbound Returns once they are picked?

Yeah it was copied from Inbound Shipment, and we should probably stay consistent with that... but you're right we would never actually see it in `New` status. Maybe you want to see the date that the Outbound Return was created vs when it was picked?